ngraph/core/src/coordinate_transform.hpp

```diff
@@ -0,0 +1,179 @@
+//*****************************************************************************
```

Comment:
is it copied? can we move to library with references?

Reply:
Fixed. Copy-past issue